src/runtime-rs/crates/resource/src/network/policy_rule.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use agent::{IPFamily, Rule};
+use anyhow::{Context, Result};
+use futures::stream::TryStreamExt;
+use netlink_packet_route::rule::RuleMessage;
+use std::net::IpAddr;
+
+pub(crate) struct NetworkRule {
```

Comment:
I think this file could be put under `util` dir

Reply:
Thx, good idea